apps/coscrad-frontend/src/components/live/live.tsx

```diff
@@ -0,0 +1,12 @@
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+
+export interface AboutProps {
+    about: string;
+}
+
+export const Live = (): JSX.Element => {
+    const { listenLive } = useContext(ConfigurableContentContext);
+
+    return <div>{listenLive}</div>;
```

Comment:
This is where you should render the icecast stream. That means you need not only the 'listenLiveMessage' but also an `iceCastStreamUrl` and maybe a `tagLine` and `radioLogoUrl`.

Reply:
this has been fixed